packages/site/cms-site/src/sitePreview/SitePreviewUtils.ts

```diff
@@ -17,7 +17,7 @@ export type SitePreviewParams = {
     previewData?: SitePreviewData;
 };
 
-export async function sitePreviewRoute(request: NextRequest, _graphQLFetch: unknown /* deprecated: remove argument in v8 */) {
+export async function sitePreviewRoute(request: NextRequest) {
```

Comment:
i would also prefer and recommand to have an codemod for removing of this argument.

Reply:
Here you go: https://github.com/vivid-planet/comet-upgrade/pull/33